### PR TITLE
Map fixes+RPD Changes

### PR DESCRIPTION
**Pull request by RadiantFlash7**
Changes to RPD spawn/rnd addition.
Map fixes related to access, areas, and some power lines!

### CODE REVIEW

**Comment by izac112** [comment]
In case you need it for debugging
```
## UNIT_TEST: --------------- Cryogaia Compound Gate(/area/borealis2/outdoors/grounds/entrance) lacks an Air scrubber.
## UNIT_TEST: --------------- Cryogaia Compound Gate(/area/borealis2/outdoors/grounds/entrance) lacks an Air vent.
## UNIT_TEST: !! FAILURE !! [MAP: Area Test APC / Scrubbers / Vents Z level 1]: [1/266]Some areas lacked APCs, Air Scrubbers, or Air vents.
```

**Comment by RadiantFlash7** [comment]
AH forgot to add that to the do not test area

File: code/modules/research/designs/engineering.dm [hunk 1]
```diff
@@ -31,6 +31,14 @@
...
 	sort_string = "NAAAC"
 
+/datum/design/item/tool/rpd
+	name = "Rapid Piping Device (RPD)"
+	desc = "A device used to rapidly pipe things."
+	id = "RPD"
+	req_tech = list(TECH_ENGINEERING = 3, TECH_MATERIAL = 2, TECH_BLUESPACE = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 300, "silver" = 100, "diamond" = 500)
+	build_path = /obj/item/weapon/pipe_dispenser
+	sort_string = "NAAAB"
```
Comment (Repede) [inline code/modules/research/designs/engineering.dm]: Add //YW addition and //YW ends or put this datum in engineering_yw.dm 😄

Comment (RadiantFlash7) [reply code/modules/research/designs/engineering.dm]: ;~; fiiiiine. It’ll be like a week before I can. Am busy.

Comment (izac112) [reply code/modules/research/designs/engineering.dm]: I can fork your code and push it to a branch on this rep and then add the comments, then you don't have to worry about it.

Comment (izac112) [reply code/modules/research/designs/engineering.dm]: or to make it easier just merge it and then i or someone can add the comments

File: code/modules/research/designs/engineering.dm [hunk 2]
```diff
@@ -31,6 +31,14 @@
 	build_path = /obj/item/weapon/tool/crowbar/power
 	sort_string = "NAAAC"
 
+/datum/design/item/tool/rpd
```
Comment (Shadow-Quill) [inline code/modules/research/designs/engineering.dm]:
```suggestion
/datum/design/item/tool/rpd // YW addition begins
```

File: code/modules/research/designs/engineering.dm (hunk 1 above)
Comment (Shadow-Quill) [inline code/modules/research/designs/engineering.dm]:
```suggestion
	sort_string = "NAAAB" // YW addition end
```

File: code/modules/research/designs/engineering.dm (hunk 2 above)
Comment (Shadow-Quill) [reply code/modules/research/designs/engineering.dm]: @RadiantFlash7 @izac112 *Or* someone can suggest changes and have Radiant accept them. \:3

**Pull request merged**